README.md

```diff
@@ -1,9 +1,45 @@
 # PHP Releases Action
-Docker container action to consume the PHP Releases API for use within GitHub Actions.
+Generates an array of supported PHP releases for use within a GitHub Actions matrix.
 
-## Requirements
+## Usage
+Within your workflow `jobs` block, add the following job _before_ your matrix job:
+```yaml
+  output_releases:
+    name: Generate PHP Releases Array
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch Current Releases
+        uses: tighten/phpreleases-action@v1
+        id: releases
+```
```

Comment:
I've seen usage details in other docs for actions so I'm suggesting we move them to match.

```suggestion
## Usage
```yaml
  # This job will need to run before the job that defines the matrix.
  output_releases:
    name: Generate PHP Releases Array
	  # Requires a machine that can execute bash and make http requests.
    runs-on: ubuntu-latest
    steps:
      - name: Fetch Current Releases
        uses: tighten/phpreleases-action@v1
        id: releases
```
```